crates/bevy_ecs/src/query/par_iter.rs

```diff
@@ -0,0 +1,133 @@
+use crate::world::World;
+use bevy_tasks::ComputeTaskPool;
+
+use super::{Fetch, QueryFetch, QueryItem, QueryState, ROQueryFetch, ROQueryItem, WorldQuery};
+
+const DEFAULT_BATCHES_PER_THREAD: usize = 4;
```

Comment:
```suggestion
const BATCH_MULTIPLIER_PER_THREAD: usize = 4;
```

Reply:
probably deserves a comment explaining what changing this does.

Reply:
No longer relevant as this is no longer a constant, this is configurable, and defaults to 1. Should be better documented too.